lib/manager/travis/readme.md

```diff
@@ -1 +1,28 @@
-This manager is intended to keep Travis config files (`.travis.yml`) up-to-date. Currently it manages only the `node_js` section of files only.
+This manager is intended to keep Travis config files (`.travis.yml`) up-to-date.
+Currently it manages only the `node_js` section of files only.
+
+An important limitation is that Renovate does not currently "understand" Travis's node matrix concept, so it will try to update all found node versions to the latest LTS, e.g.
```

Comment:
```suggestion
An important limitation is that Renovate does not currently "understand" Travis's node matrix concept, so it will try to update all found Node.js versions to the latest LTS, e.g.
```

Might want to use the proper casing for the proper noun here?

Reply:
```suggestion
An important limitation is that Renovate does not currently "understand" [Travis's Build Matrix concept](https://docs.travis-ci.com/user/build-matrix/#matrix-expansion), so it will try to update all found Node.js versions to the latest LTS, e.g.
```

By the way, it seems Travis calls it a Build Matrix, but we call it `node matrix` or `version matrix`. So we might want to settle on one term, and use that one throughout?

In addition, we might also want to link to the Travis.com docs for this build matrix feature? This is optional, so you can choose which suggestion you want to apply.